tests/fixtures/plugins/october/tester/Plugin.php

```diff
@@ -61,4 +61,14 @@ public function registerNavigation()
             ]
         ];
     }
+
+    public function registerValidationRules()
+    {
+        return [
+            'uppercase' => function ($attribute, $value, $parameters, $validator) {
+                return strtoupper($value) == $value;
```

Comment:
```suggestion
                return strtoupper($value) === $value;
```
Any reason why you used the weak comparison operator instead of the strict one or was it just habit?

Reply:
Just a habit @LukeTowers

Reply:
@SaifurRahmanMohsin in general I would recommend that you reverse that habit, `===` is going to save you from a lot of bugs that `==` can accidentally lead to.